packages/ckeditor5-dev-env/lib/release-tools/utils/packagestorelease.js

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
It took me a minute to decode this name :D I thought it's "packages to release" or "package store release" :D